lib/full-site-editing/template-loader.php

```diff
@@ -17,7 +17,9 @@ function gutenberg_add_template_loader_filters() {
 		if ( 'embed' === $template_type ) { // Skip 'embed' for now because it is not a regular template type.
 			continue;
 		}
-		add_filter( str_replace( '-', '', $template_type ) . '_template', 'gutenberg_override_query_template', 20, 3 );
+		if ( ! file_exists( get_stylesheet_directory() . '/' . $template_type . '.php' ) || null !== _gutenberg_get_template_file( 'wp_template', $template_type ) ) {
+			add_filter( str_replace( '-', '', $template_type ) . '_template', 'gutenberg_override_query_template', 20, 3 );
+		}
```

Comment:
I haven't tested it yet, but won't this break priorities for parent themes? Meaning, if a parent theme has both a PHP & a block template, will the block template still override the PHP one as it should?

Reply:
this is only checking on the child theme because of `get_stylesheet_directory`. If a parent theme has page.php and page.html, and the child defines neither, it will load page.html

Reply:
But `get_stylesheet_directory` doesn't only work for child themes.... If there is no child theme, it will fallback to the parent theme. That's what confuses me a bit here. Looking at the code it seems like if there is no child theme and we only have a "parent" theme, it will load PHP instead of HTML. :thinking: 
I'll need to test later, but we may have to add an `is_child_theme()` check

Reply:
ah I see what you mean. Yeah, that's true, this check doesn't take that into account, but I believe that's checked later, here: https://github.com/WordPress/gutenberg/blob/c41b7299d654909130a64b1336063a3a7ee5ec4c/lib/full-site-editing/template-loader.php#L68 This was only failing for child themes, parent themes were doing it correctly before this PR

Reply:
Gotcha :bulb: 
I had some more time to investigate & test the PR now, all good :+1: